src/options.c

```diff
@@ -386,6 +385,7 @@ void optionsParse(int argc, char *argv[])
     };
     int optch = 0;
     const char *errmsg;
+    int FFlagSet = 0;
```

Comment:
Change it to `bool` and `false`

Reply:
We have a mixture of bool and int in the code, better decide which will be used and explain it in the CONTRIBUTING.md at some point. I don't like bool but I'd be indifferent here just to have one project that uses it and get some practice.
https://github.com/resurrecting-open-source-projects/scrot/blob/7ee199cd9729b8887ee5574ec899374226d18bda/src/options.c#L440-L454